traffic_ops/testing/api/v5/regions_test.go

```diff
@@ -298,7 +294,7 @@ func validateRegionsDescSort() utils.CkReqFunc {
 
 func validateRegionsPagination(paginationParam string) utils.CkReqFunc {
 	return func(t *testing.T, _ toclientlib.ReqInf, resp interface{}, _ tc.Alerts, _ error) {
-		paginationResp := resp.([]tc.Region)
+		paginationResp := resp.([]tc.RegionV5)
```

Comment:
This line panics for me. You're typecasting to `[]tc.RegionV5`, which is not the return type of the `Response` field:

https://github.com/apache/trafficcontrol/blob/0d79728bbd9d12e8464a294bb33409ca2282c68c/lib/go-tc/regions.go#L44-L45